groth16/src/snark.rs

```diff
@@ -0,0 +1,36 @@
+use algebraic::errors::Result;
+use franklin_crypto::bellman::pairing::ff::PrimeField;
+use rand::Rng;
+/// The basic functionality for a SNARK.
+pub trait SNARK<F: PrimeField> {
```

Comment:
As the groth16 and plonky are seperated into two crates, seems this trait should be remove.

Reply:
Done